### ISSUE TITLE
Configure Problem

### ISSUE THREAD

**Issue by ghost** [ApolloRIP/Ocelot#3]
I have Debian 9 installed. I'm trying to configure Ocelot 1.0 I run the command:

 `./configure --with-boost-libdir=/usr/lib/x86_64-linux-gnu` 

It fails. I am not sure whether this is because I am using Debian 9 and I have spent a considerable amount of time trying to figure it out. The full output is below

```
root@312cuck:/var/www/ocelot# ./configure --with-boost-libdir=/usr/lib/x86_64-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for g++... g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking for style of include used by make... GNU
checking dependency style of g++... gcc3
checking whether C++ compiler accepts -std=c++11... yes
checking whether C++ compiler accepts -fstack-protector-strong... yes
checking for boostlib >= 1.37... yes
checking for gcc... gcc
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking whether the Boost::IOStreams library is available... yes
checking for exit in -lboost_iostreams... yes
checking build system type... x86_64-unknown-linux-gnu
checking whether the Boost::System library is available... yes
checking for exit in -lboost_system... yes
checking host system type... x86_64-unknown-linux-gnu
checking if compiler needs -Werror to reject unknown flags... no
checking for the pthreads library -lpthreads... no
checking whether pthreads work without any flags... no
checking whether pthreads work with -Kthread... no
checking whether pthreads work with -kthread... no
checking for the pthreads library -llthread... no
checking whether pthreads work with -pthread... yes
checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE
checking if more special flags are required for pthreads... no
checking for PTHREAD_PRIO_INHERIT... yes
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for libev library location... no
configure: error: Didn't find the libev library dir in '/usr/local/libev/lib64 /usr/local/libev/lib /usr/local/lib64/libev /usr/local/lib/libev /opt/libev/lib64 /opt/libev/lib /usr/lib64/libev /usr/lib/libev /usr/local/lib64 /usr/local/lib /usr/lib64 /usr/lib'
```

Any help and/or advice you can provide is appreciated. Thank you

**Comment by ligh7s** [comment]
https://askubuntu.com/questions/361741/what-are-dependencies  
tl;dr dependencies are not required for no reason

**Comment by ghost** [comment]
Thank you and which  _alleged_  dependencies am I missing here in your expert opinion?

`libev-dev is already the newest version (1:4.22-1+b1)`
`build-essential is already the newest version (12.3)`

**List of all my installed packages:**

[https://hastebin.com/ecenehemin.css](url)

**Comment by itismadness** [comment]
Looks like you're missing [libev](http://software.schmorp.de/pkg/libev.html) which is listed as [libev-dev](https://packages.debian.org/en/stretch/libev-dev) in Debian 9.

All dependencies are listed in the readme, except for [libspdlog-dev](https://packages.debian.org/en/stretch/libspdlog-dev).

**Comment by ghost** [comment]
itismadness thank you for taking the time to answer.
